### PR TITLE
Allow for optional tracking from the CDN version

### PR DESCRIPTION
**Pull request by mmcc**

### CODE REVIEW

File: src/js/cdn.js [hunk 1]
```diff
@@ -9,6 +9,10 @@
  * @type {Image}
  */
 ;(function(i,w,n,e,l){
+  // Allow people to disable CDN tracking themselves by specifying a global
+  // (VIDEOJS_NO_ANALYTICS) as true
+  if (window.VIDEOJS_NO_ANALYTICS) return;
```
Comment (heff) [inline src/js/cdn.js]: use `w` instead of `window` here

**Comment by mmcc** [comment]
@heff change made

**Comment by heff** [comment]
In the es6 branch this file has moved, but this should be merged. :+1:

**Comment by heff** [comment]
@mmcc this needs to be updated for 5.0.

**Comment by heff** [comment]
Moving this issue to the CDN repo, videojs/cdn#1. We'll be moving all of the CDN-specific logic to this repo.

**Pull request closed without merging**